tests/experimental/openai/test_proxy_workflow.py

```
@@ -0,0 +1,131 @@
+        del data, kwargs
+        return None
+
+
+class _ResponseLossMaskedRewardAgent:
+    async def run(self, data, **kwargs):
+        del data, kwargs
+        return ResponseLossMaskedReward(0.0)
+
+
+class _FakeProxyClient:
+    instances: list[_FakeProxyClient] = []
```

Ruff (0.15.12)

[warning] Mutable default value for class attribute

(RUF012)